app/stac_api/models.py

```diff
@@ -248,14 +252,14 @@ def update_extent(self, item_properties_datetime):
     def clean(self):
         # very simple validation, raises error when geoadmin_variant strings contain special
         # characters or umlaut.
-
-        for variant in self.summaries["geoadmin:variant"]:
-            if not bool(re.search('^[a-zA-Z0-9]*$', variant)):
-                logger.error(
-                    "Property geoadmin:variant not compatible with the naming conventions."
-                )
-                raise ValidationError(_("Property geoadmin:variant not compatible with the"
-                                        "naming conventions."))
+        if self.summaries["geoadmin:variant"]: # TODO: quick and dirty fix
+            for variant in self.summaries["geoadmin:variant"]:
+                if not bool(re.search('^[a-zA-Z0-9]*$', variant)):
+                    logger.error(
+                        "Property geoadmin:variant not compatible with the naming conventions."
+                    )
+                    raise ValidationError(_("Property geoadmin:variant not compatible with the"
+                                            "naming conventions."))
```

Comment:
I've fixed this in PR #58 where the `self.summaries["geoadmin:variant"]` is then an empty list per default, so I would remove this.